corehq/form_processor/models.py

```diff
@@ -42,13 +42,15 @@ class AttachmentMixin(SaveStateMixin):
     ATTACHMENTS_RELATED_NAME = 'attachments'
 
     def get_attachment(self, attachment_name):
+        return self.get_attachment_meta(attachment_name).read_content()
+
+    def get_attachment_meta(self, attachment_name):
         if hasattr(self, 'unsaved_attachments'):
             for attachment in self.unsaved_attachments:
                 if attachment.name == attachment_name:
-                    return attachment.read_content()
+                    return attachment
```

Comment:
looks like this might sometimes never return (if there are no attachments with `attachment_name`). Might want to put calls to this in `try...catch` blocks, or deal with the `None` case specifically. Unless you are sure there will always be one with `attachment_name`?